packages/legacy-compat/src/builders/find-all.ts

```diff
@@ -3,24 +3,26 @@
  */
 import { assert } from '@ember/debug';
 
-import type { StoreRequestInput } from '@ember-data/store';
-import type { FindAllOptions } from '@ember-data/store/-types/q/store';
-import type { TypedRecordInstance, TypeFromInstance } from '@warp-drive/core-types/record';
-import { SkipCache } from '@warp-drive/core-types/request';
-import type { RequestSignature } from '@warp-drive/core-types/symbols';
-
+import type Model from '@ember-data/model';
+import { SkipCache } from '@ember-data/request';
+import type { ImmutableRequestInfo } from '@ember-data/request/-private/types';
 import { normalizeModelName } from './utils';
 
-type FindAllRequestInput<T extends string = string, RT = unknown[]> = StoreRequestInput & {
+// Keeping unused generics for consistency with 5x types
```

Comment:
I know we aren't publishing these types, but removing them was more work than just keeping them so....